### PR TITLE
add movie local data source

### PR DESCRIPTION
**Pull request by kpt96**
add movie local data source
Link redmine: https://edu-redmine.framgia.vn/issues/1207

### CODE REVIEW

**Comment by kpt96** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            values.put(DBHelper.COLLUM_ID, movie.getId());
+            values.put(DBHelper.COLLUM_TITLE, movie.getTitle());
+            values.put(DBHelper.COLLUM_VOTE_AVERAGE, movie.getVoteAverage());
+            values.put(DBHelper.COLLUM_POSTER_PATH, movie.getPosterPath());
+            values.put(DBHelper.COLLUM_RELEASE_DATE, movie.getReleaseDate());
+            db.insert(DBHelper.TABLE_NAME, null, values);
+            db.close();
+            return true;
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java]: handle Ex

File: app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    public boolean deleteMovie(Movie movie) {
+        SQLiteDatabase db = mDBHelper.getWritableDatabase();
+        try {
+            db.delete(DBHelper.TABLE_NAME,
+                    DBHelper.COLLUM_ID + STRING_QUERY,
+                    new String[]{String.valueOf(movie.getId())});
+            db.close();
+            return true;
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java]: handle Ex

**Comment by kpt96** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        }
+        return false;
+    }
+
+    @Override
+    public boolean deleteMovie(Movie movie) {
+        SQLiteDatabase db = mDBHelper.getWritableDatabase();
+        try {
+            db.delete(DBHelper.TABLE_NAME,
+                    DBHelper.COLLUM_ID + STRING_QUERY,
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java]: như trên

File: app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+                    new String[]{String.valueOf(movie.getId())});
+            db.close();
+            return true;
+        } catch (SQLiteException e) {
+            e.printStackTrace();
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/moviedb_23/data/source/local/sqlite/MovieDAOImpl.java]: handle Exception

**Comment by kpt96** [comment]
ready

**Comment by kpt96** [comment]
ready

**Pull request merged**